rpi-gpio.js

```diff
@@ -181,7 +181,14 @@ function Gpio() {
                 exportPin(pinForSetup, next);
             },
             function(next) {
-                setEdge(pinForSetup, edge, next);
+              async.retry({times: 10, interval: 100},
+                function(cb){
+                  setEdge(pinForSetup, edge, cb);
```

Comment:
Sorta... in a sense, what we're really trying to create is a `writeFileUntilPermsCatchUp()` function that both `setEdge` and `setDirection` rely on. The problem we're solving is a low-level race in the OS subsystems, and not a high-level race in the gpio's logical operation.

However, given how short this library is, I don't feel strongly about this point at all.